### ISSUE TITLE
Redux store persistance

### ISSUE THREAD

**Issue by zedtux** [supnate/rekit#175]
When I'm changing the app location (through a link for instance), I see that the redux's store is not persisted.

I'm using the polyglot package, which build the `polyglot` node where the `locale` and more are stored, but when I click a link to change the location, I see that the node values are empty while before changing the location I had, at least, the locale stored.

It seems that I need something like `redux-persist`, but I'm wondering if I don't have an issue as I don't see an app built with Rekit without this feature.

Am I doing something wrong?

**Comment by supnate** [comment]
Did you use 'Link' component from 'react-redux-dom' for the navigation? Which doesn't actually refresh the page.

**Comment by zedtux** [comment]
Nope, I'm using reactstrap to build the UI, in this case a `NavItem`.

OK so this makes my app to refresh. So when building an app with Rekit studio, by default, there isn't a persisting strategy, on page refresh the Redux store is cleared?

**Comment by zedtux** [comment]
BTW I found the usage of `history.push()` from the source of the `Link` component from the `react-redux-dom` package.

I'm using it as much as I can.

But that doesn't change the purpose of this issue. Persisting the store should be implemented or at least documented.

**Comment by ianitsky** [comment]
@zedtux To persist your data you can use another approach, like localStorage.

I think that redux used to storage the data after a full refresh page.

I have a example:

```javascript
actions
    .login(data)
    .then((res) => {
      localStorage.setItem('user_data', JSON.stringify(res.data));
      this.props.history.push('/lists');
    })
    .catch((error) => {
      const inputsError = processRequestErrors(error.response.data);
      this.setState({ inputsError: inputsError });
    });
```


Now I can get the user_data after a refresh or if user close and reopen my site.

**Comment by zedtux** [comment]
I'm using the `redux-persist` package which does this for me under the hood, but thank you @ianitsky for your comment.

Just to be sure : I'm not looking for a solution to persist the data, I'm opening a discussion about persisting the data as it isn't the case by default in Rekit.

**Comment by ianitsky** [comment]
@zedtux , Thanks for the tip. I'll try redux-persist
